ts/nni_manager/training_service/reusable/aml/amlClient.ts

```diff
@@ -135,11 +135,19 @@ export class AMLClient {
     
     // Monitor error information in aml python shell client
     private monitorError(pythonShellClient: PythonShell, deferred: Deferred<any>): void {
+        pythonShellClient.on('stderr', function (error: any) {
+            // FIXME: The error will only appear in console.
+            // Still need to find a way to put them into logs.
+            console.error(`Python process stderr: ${error}`);
+        });
         pythonShellClient.on('error', function (error: any) {
```

Comment:
This guarantees to report an `Error`.
https://github.com/extrabacon/python-shell/blob/master/index.ts#L456
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/globals.d.ts#L193